articles/node-js-vs-django/index.md

```diff
@@ -0,0 +1,83 @@
+
+This article will analyze these two popular technologies. The article will critically give each member the pros and cons to help the reader choose the best technology depending on the project needs.
+
+### Node.js
+
+It is an open-source JavaScript run time. The first version was released in the [year 2009](https://nodejs.dev/learn/a-brief-history-of-nodejs). Node.js is based on the following programming languages; C, C++, and JavaScript. Node.js application is on the client-side browser. It provides scalability and concurrency.
+
+#### Basic features of Node.js
+
+1. Simplicity – It is easy to learn and use and especially with JavaScript knowledge.
+2. Open-source language – It does not require installation fees and is free to use and readily available to the public.
+3. Object-oriented – It can model real-world data rather than functions and codes.
+4. It is swift to execute codes.
+5. A lot of community tutorials, resources, and shared codes are available.
+
+### Django
+
+It is an open-source Python web framework primarily designed to ensure high-speed web development. It is easy to learn for those knowledgeable in Python. Django has strict compliance with the rules and formalized scenarios for project implementation. Django provides accuracy in coding, a straight-forward app architecture, fewer errors, and bugs in the output.
+
+#### Basic features of Django
+
+1. Simplicity – Especially with the Python language, Django is easy to learn, use, and implement.
+2. It is an open-source language – No fees for installation and usage. It is available and freely accessible to the public.
+3. High-level language – It simplifies computer syntax into a language the user can easily understand.
+4. It is both object-oriented and functional – meaning it can model real-world data and focus on functions. Node.js allows code re-useability.
+5. Portable – There is no need to write different codes for different machines as it is highly portable and flexible.
+6. It is available with a vast collection of libraries – Libraries are inbuilt, and therefore, there is no need to write unique code for everything.
+
+### Comparison between Django and Node.js
+
+| Distinctive point | Django | Node.js |
+| --- | --- | --- |
+| Programming language and primary use | It is a [Python based framework](https://www.upgrad.com/blog/python-frameworks/) used by developers to build computer apps within a stipulated time frame.| A framework that runs on JavaScript. Developers use it to build robust application programming interfaces (client-sided). |
+| Architecture | Follows the [MTV model template](https://www.javatpoint.com/django-mvt) for purposes of handling data, validating, and interaction. | A runtime environment, therefore, works on an [event-driven model](https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture). Runs on any operating system, maintaining a small number of requests. |
```

Comment:
MVT model*

Reply:
You need to explain the architecture. Do so in a separate section before you jump into the comparison.